### PR TITLE
Add Post model to the admin UI 

### PR DESCRIPTION
**Pull request by igorkruglyak**
ref https://github.com/kpi-web-guild/django-girls-blog-igorkruglyak/issues/19

### CODE REVIEW

File: blog/admin.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 """Configuration for admin Blog app."""
-# from django.contrib import admin
+from django.contrib import admin
+from .models import Post
```
Comment (webknjaz) [inline blog/admin.py:4]: Plz separate project imports from the framework imports with one empty line.

Comment (igorkruglyak) [reply blog/admin.py:4]: ok, thanks

**Comment by webknjaz** [comment]
Why do you want to autoclose #19 on merge?

**Comment by igorkruglyak** [comment]
> Why do you want to autoclose #19 on merge?

I was wrong. I wanted to insert a link to issue.

**Review by webknjaz — APPROVED** [review]

**Pull request merged**